app/models/task.py

```diff
@@ -3,4 +3,33 @@
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String, nullable=False)
+    description = db.Column(db.String, nullable=False)
+    completed_at= db.Column(db.DateTime, nullable=True)
+    # one goal has many tasks - this model is the child model
+    # we place the foreign key on the child model refering the parent
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.goal_id'), nullable=True)
+    goal = db.relationship("Goal", back_populates="tasks")
+
+
+    def to_dict(self):
+
+        value = False
+        return {
+            "id": self.task_id,
+            "title": self.title,
+            "description": self.description,
+            "is_complete": True if self.completed_at else value
+        }
```

Comment:
This reads more clearly if you don't assign `False` to a variable:

```suggestion
        return {
            "id": self.task_id,
            "title": self.title,
            "description": self.description,
            "is_complete": True if self.completed_at else False
        }
```